sdk/keyvault/azure-security-keyvault-certificates/test/samples/certificate-basic-operations/CMakeLists.txt

```diff
@@ -11,9 +11,10 @@ add_executable (
   certificate-basic-operations
   certificate_basic_operations.cpp
 )
+
 create_per_service_target_build(certificate-basic-operations keyvault)
 
-target_link_libraries(certificate-basic-operations PRIVATE azure-security-keyvault-certificates azure-identity )
+target_link_libraries(certificate-basic-operations PRIVATE azure-security-keyvault-certificates azure-identity get-env-helper)
```

Comment:
This would currently only work by building project from root directory.

We need support for building from inside a package folder ( where get-env-helper needs to be pulled as a dependency )

What if we follow what we do for `gtest` and create a new repository for `get-env` and use `fetch-content` with cmake to get the library within the build folder.

I wouldn't expect this library to keep changing often.  I think it would be better than keeping it as part of the Azure SDK for C++ repo.  At least if would be easier to integrate for all building modes ( from root or from independent service's package)

Reply:
I agree, being part of Azure SDK public API would make things simpler in many aspects, and I don't think it will add additional support cost: our code uses it anyway! And I don't expect many modifications there.

Reply:
I would make it public but limited to read only AZURE-known env variables ( any prefixed with AZURE or a list of hard coded env vars that some service is using w/o prefix)

Reply:
It will be hard to read and maintain, for little benefit.